.storybook/main.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const { getRules } = require('../scripts/build/webpack.config');
+
+module.exports = {
+    stories: ['../stories/**/*.@(js|jsx|ts|tsx)'],
```

Comment:
In the old version loading files was implemented via `index.ts` and we had a right overriding because stories were ordered in the right way in the `index.ts`. In the current implementation we still can control order by items in the array `stories`, but first we need to group our stories into subgroups/subfolders `Concepts`, `Theme`, `Atoms` etc (in the same way as we have for preview.js), for example:
```
 stories: ['../stories/theme/*.@(js|jsx|ts|tsx)', '../stories/**/*.@(js|jsx|ts|tsx)'],
```
It fixed the issue in Pagination story:
<img width="751" alt="Screenshot 2022-08-19 at 13 19 22" src="https://user-images.githubusercontent.com/19436304/185607885-26f19325-7d2d-46d2-8687-41875dc0cbe5.png">

Reply:
I'm sure I tried this before, and it didn't work. But you are right, and it does work. Nice one!!!
I'll push a commit with this.

Reply:
done

Reply:
We still have a problem with overriding styles for `Pagination`, because these styles come from `Themeroller` that comes after `Pagination`, we need to put it on before high-level components:
```
    stories: [
        '../stories/theme/*.@(js|jsx|ts|tsx)',
        '../stories/atoms/*.@(js|jsx|ts|tsx)',
        '../stories/molecules/*.@(js|jsx|ts|tsx)',
        '../stories/organisms/*.@(js|jsx|ts|tsx)',
        '../stories/packages/*.@(js|jsx|ts|tsx)',
        '../stories/**/*.@(js|jsx|ts|tsx)',
    ],
```

Reply:
I'm actually not convinced that the theme should be at the beginning. Its whole purpose is to overwrite the styles, and then storybook will showcase the components as they will look like if adding that theme to the application. What do you think?

Reply:
done